.travis.yml

```diff
@@ -3,10 +3,12 @@ python:
     - "3.4"
 cache: "pip"
 install:
-    - "pip install ."
+    - "pip install .[tests]"
 script:
     - "flake8 ."
```

Comment:
Not in the scope of this PR, but maybe later: Consider using pytest-flake8 here.